src/components/styledSelectOption/styledSelectOption.module.css

```diff
@@ -14,9 +14,3 @@
 .root:last-of-type {
   border-bottom: none;
 }
-
-@media (min-width: 769px) {
-  .root {
-    font-size: 1.025rem;
```

Comment:
This was just kind of a fly in the ointment - minimal visible difference but messed with the truncation.